quic/s2n-quic-transport/src/space/application.rs

```diff
@@ -1014,3 +1143,41 @@ impl<Config: endpoint::Config> PacketSpace<Config> for ApplicationSpace<Config>
         Ok(())
     }
 }
+
+#[derive(Default)]
+struct SkippedPacketNumber {
+    // skipped for PTO probe
+    pto: Option<PacketNumber>,
+    // skipped for Optimistic Ack mitigation
+    opt_ack: Option<PacketNumber>,
+}
+
+#[cfg(any(test, feature = "testing"))]
+mod tests {
+    use super::*;
+    use crate::path::testing::helper_path_server;
+    use bolero::check;
+    use s2n_quic_core::random;
+
+    #[test]
+    fn fuzz_arm_skip_counter() {
+        let mut skip_counter = None;
+
+        let mut path = helper_path_server();
+        assert_eq!(path.mtu(transmission::Mode::Normal), 1200);
+        let mtu = path.mtu(transmission::Mode::Normal) as u32;
+
+        check!().with_type().cloned().for_each(|cwnd| {
+            let random = &mut random::testing::Generator::default();
```

Comment:
this will have the same seed on every call. ideally you seed the generator from fuzzer input

https://github.com/aws/s2n-quic/blob/6056fd4c5790f8d0ab000139cfbd6d8d8e6197e7/quic/s2n-quic-core/src/random.rs#L47